resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -119,25 +119,25 @@ function createListElement(date, type, text, id, author) {
   return liElement;
 }
 
-/**
- * Fetches the blobstore-serve to sends its response as an array buffer to the Adobe DC View
- */
-
+ //Adobe Preview configurations for getRevieweeResume function
  const previewConfig = {
   'showLeftHandPanel': true,
   'showPageControls': true,
   'showDownloadPDF': false,
-  'showAnnotationTools': true,
   'showPrintPDF': false,
   'enableAnnotationsAPI': true,
   'includePDFAnnotations': true
 }
 
+ /**
+ * Fetches the blobstore-serve to sends its response as an array buffer to the Adobe DC View
+ */
 async function getRevieweeResume() {
   fetch('/blobstore-serve')
     .then((response) => {
       const pdfId = response.headers.get('blobKeyString');
       const resumeFileName = response.headers.get('newResumeFileName');
+      const showAnnoTools = response.headers.get('annoToolBool');
```

Comment:
You aren't doing anything with this value. I think I provided some comments to show you how you would pass this value into the object stored in the `previewConfig` variable.

Reply:
Yes! I think when I was trying to pull from this branch, and from the resume buddy master something happened, because initially I did make this change. I wonder what happened.